dev/scripts/src/alluxio.org/build-distribution/cmd/generate-tarball.go

```diff
@@ -202,7 +202,7 @@ func addAdditionalFiles(srcPath, dstPath string, hadoopVersion version, version
 			"bin/alluxio-stop.sh",
 			"bin/alluxio-workers.sh",
 			"bin/launch-process",
-			fmt.Sprintf("client/alluxio-%v-client.jar", version),
+			fmt.Sprintf("client/build/alluxio-%v-hadoop2-client.jar", version),
```

Comment:
oh seems we only included the hadoop 2 jar here, and missed out hadoop 3? @Xenorith do you recall what our decision was? Thanks!

Reply:
resolved by @Kai-Zhang in https://github.com/Alluxio/alluxio/pull/18090/files